apps/mobile/src/components/ChatDetailScreen/components/MessageItem.tsx

```diff
@@ -21,11 +21,6 @@ function MessageItem({
   const {otherSideDataAtom} = useMolecule(chatMolecule)
   const {t} = useTranslation()
   const {userName, image} = useAtomValue(otherSideDataAtom)
-  if (
```

Comment:
We want to display it now. This way we can show all chat history. See below how these messages are handled.